### PR TITLE
Fixed text color in LogoutSpots

### PR DESCRIPTION
**Pull request by cattyngmd**
oops

### CODE REVIEW

File: src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java [hunk 1]
```diff
@@ -66,8 +65,7 @@ public void invoke(Render3DEvent event)
                 RenderUtil.endRender();
 
                 if (module.nametags.getValue()) {
-                    String text = TextColor.RED
-                            + spot.getName()
+                    String text = spot.getName()
```
Comment (3arthqu4ke) [inline src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java:68]: Where did the TextColor.RED go?

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java:68]: nvm are you coloring the spot

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java:68]: sec

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/render/logoutspots/ListenerRender.java:68]: Ok forge this one

**Review by 3arthqu4ke — CHANGES_REQUESTED** [review]
I missed a thing! Where you draw the spot text:
```java
module.color.getValue().hashCode()
```
Have you tested that? Afaik hashCode gives RGBA or RGB but our and Minecrafts FontRenderer takes ARGB

**Comment by 3arthqu4ke** [comment]
LGTM™ moment

**Comment by cattyngmd** [comment]
> I missed a thing! Where you draw the spot text:
> 
> ```java
> module.color.getValue().hashCode()
> ```
> 
> Have you tested that? Afaik hashCode gives RGBA or RGB but our and Minecrafts FontRenderer takes ARGB

ye it works fine

**Comment by cattyngmd** [comment]
wait

**Comment by cattyngmd** [comment]
i didnt test it with custom font renderer

**Comment by cattyngmd** [comment]
ye its fine

**Comment by 3arthqu4ke** [comment]
can u log out on cc rn

**Comment by cattyngmd** [comment]
yeah

**Comment by 3arthqu4ke** [comment]
Ok looks awesome, yeah weird, we use Color.getRGB for rendering a string somewhere else and it works, seems like its ARGB after all: `Bits 24-31 are alpha`

**Review by 3arthqu4ke — APPROVED** [review]

**Comment by 3arthqu4ke** [comment]
Thanks a lot for contributing so much! Will make a release once HoleFiller - Offhand has been tested.

**Comment by cattyngmd** [comment]
> Thanks a lot for contributing so much! Will make a release once HoleFiller - Offhand has been tested.

np :D

**Pull request merged**